content/_guides/managing-user-roles-in-a-tenant.md

```diff
@@ -33,8 +26,8 @@ The default **contract roles** are:
 The default **workspace roles** are:
 
 *   **Owner** - Users with this role can edit the workspace, edit [flows](/getting-started/integration-flow), start/stop flows, change flows from/to [real-time/ordinary](realtime-flows) status, and edit credentials in a workspace.
```

Comment:
>  edit the workspace
maybe it makes sense to clarify what exactly means `edit workspace` - rename, add/invite new members, manage members roles